README.md

```diff
@@ -36,10 +36,10 @@ To use our themes in your application, simply wrap the app in a `Provider` and p
 
 ```tsx
 import { Provider as ThemeProvider} from 'react-native-paper';
-import * as PXBThemes from '@pxblue/react-native-themes';
+import * as BluiThemes from '@brightlayer-ui/react-native-themes';
```

Comment:
Can we change BluiThemes => BLUIThemes ?

Reply:
Done.